charts/devlake/templates/ingresses.yaml

```diff
@@ -56,6 +56,7 @@ spec:
     - host: {{ .Values.ingress.hostname | quote }}
       http:
         paths:
+          {{- if not .Values.grafana.oauthEnabled }}
```

Comment:
Just asking here: Why are we not creating the internal Grafana ingress when the oauth is enabled?

Reply:
I think it's because this is how the current Config-UI/DevLake works at the moment; there is an Nginx running inside the Config-UI pod, which forwards `/grafana` traffic to the Grafana pod inside the same namespace in the K8s cluster.